smart_lamp/lamp-test.py

```diff
@@ -12,6 +12,24 @@
 def config():
     return getConfigResponse()
 
+@app.route('/serverConfig', methods=['POST'])
+def setServerConfig():
+    print(request)
+    print(request.get_json(force=True))
```

Comment:
What does the method do if the format of the body is not JSON ?

Reply:
Bug probably